global/js/main.js

```diff
@@ -190,28 +221,13 @@ function updateTheme() {
                 document.getElementById("css_cont").innerHTML += "<link id='" + theme + "_relative' rel='stylesheet' type='text/css' href='css/" + theme["css"][i] + "?v=" + version + "'>";
             }
         }
+        if (theme["customAccent"] != undefined) {

```

Comment:
its better to check undefined with typeof than directly
```suggestion
        if (typeof theme["customAccent"] != 'undefined') {

```